pyrabbit2/api.py

```diff
@@ -692,13 +692,6 @@ def get_messages(self, vhost, qname, count=1,
 
         vhost = quote(vhost, '')
         base_body = {'count': count, 'requeue': requeue, 'encoding': encoding}
-
-        # 3.7.X now uses ackmode to denote the requeuing capability
-        if requeue:
```

Comment:
I do not think getting rid of this is helpful. It breaks the 3.7.X compatibility added in #16 

Maybe you just need to rebase from master?

Reply:
Done ! that was unintentional!

Reply:
Thanks!